lib/spack/docs/getting_started.rst

```diff
@@ -643,7 +643,16 @@ Fortran.
 
 #. There are different ways to get ``gfortran`` on macOS. For example, you can
    install GCC with Spack (``spack install gcc``) or with Homebrew
-   (``brew install gcc``).
+   (``brew install gcc``). To avoid compiling all of ``gcc``, users may obtain
+   a precompiled binary `prepared by the developers
+   <https://gcc.gnu.org/wiki/GFortranBinaries#MacOS>`_. For a given `release
+   installer <https://github.com/fxcoudert/gfortran-for-macOS/releases>`_,
+   download the ``.dmg`` binary, double click it, and once the dmg is mounted
+   right-click the ``gfortran.pkg`` file and select "open" (to bypass
+   codesigning requirements by Apple). It will install to the directory
+   ``/usr/local/gfortran/``, and produce a symbolic link of
+   ``/usr/local/bin/gfortran`` (but ``gcc`` and ``g++`` will **not** be linked
+   to ``/usr/local/bin``).
```

Comment:
I would make this considerably shorter. Something like:
```rst
#. There are different ways to get ``gfortran`` on macOS. For example, you can install GCC with Spack (``spack install gcc``), with Homebrew (``brew install gcc``), or from `DMG installer <https://github.com/fxcoudert/gfortran-for-macOS/releases>`_.
```
Then, in the next section where it discusses where to find the installed executables, you can say:
```rst
If you used Spack to install GCC, you can get the installation prefix by ``spack location -i gcc`` (this will only work if you have a single version of GCC installed). Whereas for Homebrew, GCC is installed in ``/usr/local/Cellar/gcc/x.y.z``. With the DMG installer, the correct path will be ``/usr/local``.
```

Reply:
Awesome, much better.  Added slight verbiage changes above but otherwise this reads great :slightly_smiling_face:

```console
$ readlink /usr/local/bin/gfortran 
/usr/local/gfortran/bin/gfortran

$ ls /usr/local/gfortran/bin/
cpp*        gcc*        gcc-nm*     gcov*       gcov-tool*  gfortran*
g++*        gcc-ar*     gcc-ranlib* gcov-dump*  gdc*        lto-dump*
```

Not exactly clear why they install to `gfortran` directory since it's got everything.  Initially you said to do `/usr/local`, which is correct, but I guess I feel it's probably better to let them know where it comes from since in theory (not that it is desirable in practice given the state of affairs) you could use this to obtain a full gcc toolchain on macOS?  Happy to change back to initially suggested `/usr/local`!